install/playbooks/roles/system-post-install/tasks/partitions-security.yml

```diff
@@ -4,6 +4,8 @@
 - name: Check if /tmp is a mount point
```

Comment:
Unrelated?

Reply:
Yes, definitely, I will remove it from there.

Reply:
It is a fix as it was failing when /tmp was not a mounting point.
We can remove this and place it in another branch, if you want.

Reply:
Ok, I see. At least in a separate commit, it would be ok for me.